SonarQube.Client.Tests/Requests/DefaultConfiguration_Configure.cs

```diff
@@ -34,21 +35,21 @@ public void Configure_Writes_Debug_Messages()
         {
             var logger = new TestLogger();
 
-            DefaultConfiguration.Configure(new RequestFactory(logger));
-
-            logger.DebugMessages.Should().ContainInOrder(
-                new[]
+            var expected = new string[]
```

Comment:
Previously the test didn't fail if there were unexpected requests in the list. Now it does.